src/client/App.js

```diff
@@ -31,16 +32,17 @@ function App() {
         <Route exact path="/">
           <LandingPageContainer />
         </Route>
+        <Route exact path="/order/:id">
+          <OrderPageContainer id="id" email="email" />
```

Comment:
This should be ConfirmationPage

Reply:
I think it shoud go in `AuthenticatedRoute` since you need to have currentUser email